hare/hare.go

```diff
@@ -470,7 +470,24 @@ func goodProposals(ctx context.Context, logger log.Log, msh mesh, nodeID types.N
 	if ownHdr != nil {
 		ownTickHeight = ownHdr.TickHeight()
 	}
+	atxs := map[types.ATXID]int{}
 	for _, p := range props {
+		atxs[p.AtxID]++
+	}
+	for _, p := range props {
+		if p.IsMalicious() {
+			logger.With().Warning("not voting on proposal from malicious identity",
+				log.Stringer("id", p.ID()),
+			)
+			continue
+		}
+		if atxs[p.AtxID] > 1 {
```

Comment:
```suggestion
		// Check for multiple proposals sharing a single ballot.
		if atxs[p.AtxID] > 1 {
```

Reply:
sorryy i don't want to have this message. this is a bug that i observed, it may not be the only one, and i want to have this check regardless of that bahavior